addons/website/models/website_visitor.py

```diff
@@ -117,13 +117,9 @@ def _compute_last_visited_page_id(self):
 
     @api.depends('last_connection_datetime')
     def _compute_time_statistics(self):
-        results = self.env['website.visitor'].with_context(active_test=False).search_read([('id', 'in', self.ids)], ['id', 'last_connection_datetime'])
-        mapped_data = {result['id']: result['last_connection_datetime'] for result in results}
-
         for visitor in self:
-            last_connection_date = mapped_data[visitor.id]
-            visitor.time_since_last_action = _format_time_ago(self.env, (datetime.now() - last_connection_date))
-            visitor.is_connected = (datetime.now() - last_connection_date) < timedelta(minutes=5)
+            visitor.time_since_last_action = _format_time_ago(self.env, (datetime.now() - visitor.last_connection_datetime))
```

Comment:
I'm sorry to ask again but what does it "fix" exactly?
Really sounds like performance improvement / cleaning to me.

Reply:
Indeed, this is not a fix (anymore). There was a time where RLI (RIP) did this PR : #38457

Reply:
As discussed:

Change it back to FIX and explain it prevents a stack trace when creating visitors in the backend.